docs/user_guides/fs/feature_view/transformation-function.md

```diff
@@ -1,75 +1,151 @@
+        @udf([int, int])
+        def add_one_and_two(feature1):
+            return pd.DataFrame({"add_one":feature1 + 1, "add_two":feature1 + 2})
+        ```
 
-        # get built-in transformation function min max scaler
-        min_max_scaler_fn = fs.get_transformation_function(name="min_max_scaler")
+Creation of a Many to Many transformation function is similar to that of One to May transformation function, the only difference being that the transformation function accepts multiple features as input.
 
-        # get transformation function by name and version.
-        plus_one_fn = fs.get_transformation_function(name="plus_one", version=2)
+=== "Python"    
+    !!! example "Creation of a Many to Many Custom Transformation Function in Hopsworks."
+        ```python
+        from hopsworks import udf
+        import pandas as pd
+
+        @udf([int, int, int])
+        def add_one_multiple(feature1, feature2, feature2):
+            return pd.DataFrame({"add_one_feature1":feature1 + 1, "add_one_feature2":feature2 + 1, "add_one_feature3":feature3 + 1})
+        ```
+To access statistics pertaining to an argument provided as input to the transformation function, it is necessary to define a keyword argument named `statistics` in the transformation function. This statistics argument should be provided with an instance of class `TransformationStatistics` as default value. The `TransformationStatistics` instance must be initialized with the names of the arguments for which statistical information is required.
+
+The `TransformationStatistics` instance contains separate objects with the same name as the arguments used to initialize it. These objects encapsulate statistics related to the feature as instances of the `FeatureTransformationStatistics` class. Upon instantiation, instances of `FeatureTransformationStatistics` are initialized with `None` values. These placeholders are subsequently populated with the required statistics when the training dataset is created.
+
+=== "Python"   
+    !!! example "Creation of a Custom Transformation Function in Hopsworks that accesses Feature Statistics"
+        ```python
+        from hopsworks import udf
+        from hsfs.transformation_statistics import TransformationStatistics
+
+        stats = TransformationStatistics("feature1", "feature2", "feature3") 
+
+        @udf(int)
+        def add_features(feature1, feature2, feature3, statistics=stats):
+            return feature + feature2 + feature3 + statistics.feature1.mean + statistics.feature2.mean + statistics.feature3.mean
         ```
 
+The output column generated by the transformation function follows a naming convention structured as `functionName_features_outputColumnNumber`. For instance, for the function named `add_one_multiple`, the output columns would be labeled as `add_one_multiple_feature1-feature2-feature3_0`, `add_one_multiple_feature1-feature2-feature3_1`, and `add_one_multiple_feature1-feature2-feature3_2`.
```

Comment:
`-` is not a valid character for feature name in hopsworks, probably use `_` instead. Also, is `features` the arguments name or the features name?